app/Http/Requests/StoreRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Website\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "email" => ["required", "email", "string", "max:255", "regex:/^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/"],
```

Comment:
Regex is for what?

Reply:
It's used to validate email addresses structure

Reply:
Why? Isn't `"email"` rule enough?

Reply:
Without regex but with email rule text like `aleksandra@blumilk` is passed as a valid email

Reply:
lets move regex into some const with readable variable name.